docs/lsm/index.rst

```diff
@@ -89,12 +123,19 @@
 
     The state machine attached to the lifecycle will determine whether the API call is successful or not.
 
-    TODO: Three set of attributes
+    Each service instance defines 3 sets of its attributes that are used in different situations:
+
+    - ``candidate_attributes``: The set of attributes used when the service instance model state is candidate or designed.
+    - ``active_attributes``: The set of attributes when the model is evaluated with only active instances.
```

Comment:
This is not True. These attributes are also used when a validation compile is done for another service instance for example.